js/dom.js

```diff
@@ -459,42 +462,33 @@ async function waitForSheet({
       }
     },
   };
-
-  window.on('mousedown', suppressFocusRingOnClick, {passive: true});
-  window.on('keydown', keepFocusRingOnTabbing, {passive: true});
-
+  const lazyScripts = [
+    '/js/dom-on-load',
+  ];
+  const elHtml = document.documentElement;
   if (!/^Win\d+/.test(navigator.platform)) {
-    document.documentElement.classList.add('non-windows');
+    elHtml.classList.add('non-windows');
   }
   // set language for a) CSS :lang pseudo and b) hyphenation
-  document.documentElement.setAttribute('lang', chrome.i18n.getUILanguage());
-  document.on('keypress', clickDummyLinkOnEnter);
-  document.on('wheel', changeFocusedInputOnWheel, {capture: true, passive: false});
-  document.on('click', showTooltipNote);
-
-  Promise.resolve().then(async () => {
-    if (!chrome.app) addFaviconFF();
-    await prefs.ready;
-    waitForSelector('details[data-pref]', {recur: Collapsible.bindEvents});
-  });
-
-  onDOMready().then(() => {
-    splitLongTooltips();
-    debounce(addTooltipsToEllipsized, 500);
-    window.on('resize', () => debounce(addTooltipsToEllipsized, 100));
-  });
-
-  window.on('load', () => {
-    const {sheet} = $('link[href^="global.css"]');
-    for (let i = 0, rule; (rule = sheet.cssRules[i]); i++) {
-      if (/#\\1\s?transition-suppressor/.test(rule.selectorText)) {
-        sheet.deleteRule(i);
-        break;
-      }
-    }
-  }, {once: true});
-
-  function addFaviconFF() {
+  elHtml.setAttribute('lang', chrome.i18n.getUILanguage());
+  // set up header width resizer
+  const HW = 'headerWidth.';
+  const HWprefId = HW + location.pathname.match(/^.(\w*)/)[1];
+  if (prefs.knownKeys.includes(HWprefId)) {
+    Object.assign(dom, {
+      HW,
+      HWprefId,
+      setHWProp(width) {
+        width = Math.round(Math.max(200, Math.min(innerWidth / 2, Number(width) || 0)));
```

Comment:
We can't control it via CSS because we use this variable for stuff other than `width` e.g. padding or flex-base so we would need `min` and `max` in CSS but these aren't implemented in our minimum supported browsers.